Pod/Classes/PIDatePicker.swift

```diff
@@ -490,6 +490,7 @@ public class PIDatePicker: UIView, UIPickerViewDataSource, UIPickerViewDelegate
             }
             
             self.date = self.calendar.dateFromComponents(components)!
+			sendActionForControlEvents(.ValueChanged)
```

Comment:
Can you just and `self.sendActionForControlEvents(` and left align it with `self.date`?